src/main/resources/jira/jiraIssueSearchPlugin.json

```diff
@@ -0,0 +1,4 @@
+{
+  "pluginUri": "https://raw.githubusercontent.com/blackducksoftware/blackduck-alert/10c46e3/src/main/resources/jira/atlassian-connect.json",
+  "pluginName": "Alert issue property indexer"
```

Comment:
Please change the plugin name to "Alert Issue Property Indexer"

Reply:
I don't think this or any of the JSON files here are what we want to use. I'm working on the actual plugin piece: 
https://github.com/blackducksoftware/blackduck-alert/compare/gk_jira_app

Reply:
Yes, Gavin is updating to the proper files. Mine was just a fill in for testing purposes.

Reply:
This file should be deleted once int-jira-common is released.